src/support_sphere_py/src/tests/resources/scripts/update_db_sample_data.py

```diff
@@ -0,0 +1,41 @@
+import csv, os
+from support_sphere.models.public import UserProfile, People
+from support_sphere.models.auth import User
+from support_sphere.repositories.auth import UserRepository
+from support_sphere.repositories.public import UserProfileRepository, PeopleRepository
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def populate_user_details():
+    with open("./src/support_sphere_py/src/tests/resources/data/sample_data.csv", mode='r', newline='') as file:
```

Comment:
Instead of a hardcoded path, could you make this path using `pathlib.Path`?

Reply:
updated.